Misc/NEWS.d/next/C API/2022-10-01-09-56-27.gh-issue-97588.Gvg54o.rst

```diff
@@ -0,0 +1 @@
+Fix ctypes construction of structs from description.
```

Comment:
Can you expand a bit here? This change could presumably impact anyone using packed bit fields through ctypes, so it would be good to mention at least some of those keywords here. A couple of sentences are okay (and feel free to add "`Contributed by <your name>"` at the end if you like - you've earned it 😉 )

Reply:
In fact maybe a what's new entry would be appreciated too?

Reply:
Maybe... "we're getting it right now" isn't the sort of thing we'd usually advertise as "new", but on the other hand, there's a good chance people have discovered it was buggy before and are now avoiding it. A note there may get noticed.

We still have someone edit what's new before it goes out, right? It won't hurt to add it, and the editor can choose whether to keep it or rephrase/reframe it.

For @matthiasgoergens, we're talking about `Doc/whatsnew/3.13.rst` as I mentioned for porting notes, but in the "Improved Modules" section (`ctypes` probably needs to be added as a heading).